sklearn/datasets/openml.py

```diff
@@ -536,11 +540,27 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
     # download data features, meta-info about column types
     features_list = _get_data_features(data_id, data_home)
 
+    if ignore_strings:
+        string_features = list(filter(lambda f: f['data_type'] == 'string',
+                                      features_list))
+        if string_features:
+            string_feature_names = list(map(lambda f: f['name'],
+                                            string_features))
+            warn("STRING attributes which are not yet supported. "
```

Comment:
I would not warn here. I would also think that `ignore_strings=False` should be the default since that we will have pretty quick support for dataframe

Reply:
Changed.